src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,71 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from "classnames";
+
+interface PaginationProps {
+  currentPage: number;
+  totalItems: number;
+  itemsPerPage: number;
+  onPageChange: (page: number) => void;
+}
+
+const Pagination: React.FC<PaginationProps> = ({
+  currentPage,
+  totalItems,
+  itemsPerPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(totalItems / itemsPerPage);
+  const pages = Array.from({ length: totalPages }, (_, index) => index + 1);
+
+  return (
+    <ul className="pagination">
+      <li className={classNames ('page-item' , {
+        disabled: currentPage === 1,
+      })}
+          >
+        <button
+          data-cy="prevLink"
+          className="page-link"
+          onClick={() =>
+            currentPage !== 1 ? onPageChange(currentPage - 1) : null
+          }
```

Comment:
just add `disabled` attribute and pass only function here 

https://www.w3schools.com/tags/att_button_disabled.asp